openstudiocore/src/model/AirLoopHVAC.cpp

```diff
@@ -1383,6 +1401,21 @@ namespace detail {
     return result;
   }
 
+  boost::optional<Node> AirLoopHVAC_Impl::returnAirNode() const
```

Comment:
@jmarrec here is some more stuff, what branch is this from?  is it not already on develop?